charts/kubeai/templates/configmap.yaml

```diff
@@ -12,6 +12,20 @@ data:
       {{- .Values.resourceProfiles | toYaml | nindent 6 }}
     modelServers:
       {{- .Values.modelServers | toYaml | nindent 6 }}
+    {{- if .Values.modelServerPods }}
+    modelServerPods:
+      {{- if .Values.modelServerPods.podSecurityContext }}
+      podSecurityContext:
+        {{- .Values.modelServerPods.podSecurityContext | toYaml | nindent 8}}
+      {{- end}}
+      {{- if .Values.modelServerPods.securityContext }}
+      securityContext:
+        {{- .Values.modelServerPods.securityContext | toYaml | nindent 8}}
+      {{- end}}
+      {{- if .Values.modelServiceAccount.create }}
```

Comment:
What should be the behavior if someone sets `.Values.modelServiceAccount.name` and creates this K8s SA themselves?

I think in that case the serviceAccountName should be set.

I think this if should be removed and then we need to ensure that `.Values.modelServiceAccount.name` is `default` and `create=false` by default.

Reply:
Note we do this as a follow up because create=true by default. So this is more of an edge case.

Reply:
> What should the behavior be if someone sets `.Values.modelServiceAccount.name` with `create=false` and creates this K8s SA themselves without using helm?
> 
> I think in that case the serviceAccountName should be set.
> 
> I think this if statement could be removed?

Yes, I agree. This statement is not needed.